compiler/src/iree/compiler/Codegen/Dialect/PCF/IR/PCFBase.td

```diff
@@ -0,0 +1,280 @@
+// Copyright 2025 The IREE Authors
+//
+// Licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#ifndef IREE_CODEGEN_DIALECT_PCF_DIALECT
+#define IREE_CODEGEN_DIALECT_PCF_DIALECT
+
+include "mlir/IR/DialectBase.td"
+include "mlir/IR/CommonTypeConstraints.td"
+include "iree/compiler/Codegen/Dialect/PCF/IR/PCFInterfaces.td"
+include "mlir/IR/AttrTypeBase.td"
+include "mlir/IR/BuiltinTypeInterfaces.td"
+include "mlir/IR/EnumAttr.td"
+
+//===----------------------------------------------------------------------===//
+// Parallel control flow dialect
+//===----------------------------------------------------------------------===//
+
+def PCF_Dialect : Dialect {
```

Comment:
High-level question: once we've seen this working in real code / have had it settle down, do we think we might want to upstream this?

Reply:
I'm not particularly interested in moving it upstream because that just makes it harder for us to change the dialect and gives us less control over it. Also many of the interesting interface implementations are going to inevitably be tied to IREE stuff so moving it upstream leaves it stranded.